addons/stock/models/stock_picking.py

```diff
@@ -929,6 +944,12 @@ def button_validate(self):
         pickings_to_backorder.with_context(cancel_backorder=False)._action_done()
         return True
 
+    def button_set_done_quantities(self):
```

Comment:
action_set_done_quantities button_ is the old convention we try to use action_ for public method now